### PR TITLE
Use https, mention libffi-devel

### PR DESCRIPTION
**Pull request by tyll**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -41,8 +41,8 @@ Running the Tests
 
 ::
 
-    # Create a virtualenv
-    $ sudo yum install python-virtualenv
+    # Create a virtualenv and install deps
+    $ sudo dnf install python-virtualenv libffi-devel
```
Comment (pypingou) [inline README.rst]: fedmsg_meta doesn't use libffi, neither does fedmsg

Comment (tyll) [inline README.rst]: I am not sure where it comes from, maybe from requests but I got this error without it:
```
$ python setup.py develop
running develop
running egg_info
creating fedmsg_meta_fedora_infrastructure.egg-info
writing requirements to fedmsg_meta_fedora_infrastructure.egg-info/requires.txt
writing fedmsg_meta_fedora_infrastructure.egg-info/PKG-INFO
writing top-level names to fedmsg_meta_fedora_infrastructure.egg-info/top_level.txt
writing dependency_links to fedmsg_meta_fedora_infrastructure.egg-info/dependency_links.txt
writing entry points to fedmsg_meta_fedora_infrastructure.egg-info/entry_points.txt
writing manifest file 'fedmsg_meta_fedora_infrastructure.egg-info/SOURCES.txt'
reading manifest file 'fedmsg_meta_fedora_infrastructure.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'fedmsg_meta_fedora_infrastructure.egg-info/SOURCES.txt'
running build_ext
Creating gh-tyll-fedmsg_meta_fedora_infrastructure/fedmsg_meta_fedora_infrastructure/lib/python2.7/site-packages/fedmsg-meta-fedora-infrastructure.egg-link (link to .)
Adding fedmsg-meta-fedora-infrastructure 0.17.6 to easy-install.pth file
[...]
Installed [...]gh-tyll-fedmsg_meta_fedora_infrastructure/fedmsg_meta_fedora_infrastructure/lib/python2.7/site-packages/munch-2.0.4-py2.7.egg
Searching for cryptography
Reading https://pypi.python.org/simple/cryptography/
Downloading https://pypi.python.org/packages/03/1a/60984cb85cc38c4ebdfca27b32a6df6f1914959d8790f5a349608c78be61/cryptography-1.5.2.tar.gz#md5=f977a6ddb38b071213c0157fc77c62ab
Best match: cryptography 1.5.2
Processing cryptography-1.5.2.tar.gz
Writing /tmp/easy_install-iecTqQ/cryptography-1.5.2/setup.cfg
Running cryptography-1.5.2/setup.py -q bdist_egg --dist-dir /tmp/easy_install-iecTqQ/cryptography-1.5.2/egg-dist-tmp-CJ836n
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libffi' found
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libffi' found
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libffi' found
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libffi' found
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libffi' found
c/_cffi_backend.c:15:17: schwerwiegender Fehler: ffi.h: No such file or directory
 #include <ffi.h>
                 ^
Kompilierung beendet.
error: Setup script exited with error: command 'gcc' failed with exit status 1
```

Comment (pypingou) [inline README.rst]: So it's cryptography that uses it :(

**Comment by pypingou** [comment]
:+1: for me

**Comment by tyll** [comment]
Thank you for the review, I'll merge it.

**Pull request merged**